gradle.properties

```diff
@@ -1,3 +1,4 @@
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```

Comment:
No dependencies requires Jetifier was found by Studio, I assume it's safe to disable now